set.mm

```diff
@@ -369106,6 +369106,95 @@ varying definitions (some start from 0, others start from 1), but
        renamer theorem like ~ cbval . The section
        ~ mmtheorems32.html#mm3146s also describes the
        metatheorem that underlies this.
+       </p>
+
+       <p>
+       Standard Metamath verifiers do not distinguish between axioms and
+       definitions (both are $a statements).
+       In practice, we require that definitions (1) not be creative
```

Comment:
s/creative/conservative/

Reply:
Let's do s/not be creative/be conservative/ instead :-).